package.json

```diff
@@ -43,7 +43,7 @@
     "@babel/core": "^7.12.3",
     "@babel/plugin-transform-runtime": "^7.12.1",
     "@babel/preset-env": "^7.12.1",
-    "@okta/okta-auth-js": "^5.0.0",
+    "@okta/okta-auth-js": "^5.2.3",
```

Comment:
Which part of the repo is using okta-auth-js as devDeps? the test app? We probably can move it to the submodule package.json. 

Can you create a jira to track it?

Reply:
Ies, the test app.
Internal ref: OKTA-415460